alloptions-limit.php

```diff
@@ -25,12 +26,16 @@ function run_alloptions_safeguard() {
 		return;
 	}
 
-	// Uncompressed size thresholds.
-	// Warn should *always* be =< die
-	$alloptions_size_warn = MB_IN_BYTES * 2.5;
+	$home_url_parsed = wp_parse_url( home_url() );
```

Comment:
Can we also check that it's a production environment before alerting? We can use the `VIP_GO_APP_ENVIRONMENT` constant

Reply:
https://github.com/Automattic/vip-go-mu-plugins/blob/16236fd34061ebbc6501e2c7fb59d40e33c37ef6/alloptions-limit.php#L134